### ISSUE TITLE
neon-cluster-operator: wrap-up issues

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1901]
I've verified that cluster stabilization works and I've coded the neon-cluster-operator job but couldn't test that due to: https://github.com/nforgeio/neonKUBE/issues/1900

**Comment by jefflill** [comment]
So the scheduling problem with the terminated pod GC job was due to a mistyped CRON schedule, which I've fixed.

The other thing that's a bit odd with the current design is that the job classes all reference `CronJob` as the base class to pick up the `Name,` Group`, and `Type` properties as well as the `AddToSchedulerAsync()` and `DeleteFromSchedulerAsync()` methods.  The current design has us create job instances that are never actually executed but instead are used for managing job schedules.

This seems like a bit of anti-pattern and in fact, I added `TerminatedPodGcDelayMilliseconds` and `TerminatedPodGcThresholdMinutes` properties to the `TerminatedPodGcJob` and set those on the global job, thinking that instance was going to be executed.  After playing with Quartz a bit, I see that Quartz constructs a new job instance every time a job is executed, so `TerminatedPodGcDelayMilliseconds/TerminatedPodGcThresholdMinutes` will never be set the way I coded things.

I'm going to refactor this by removing the `CronJob` base class and relocating the scheduling code as well as making the `TerminatedPodGcDelayMilliseconds/TerminatedPodGcThresholdMinutes` properties **static**.  This will seem less confusing and more natural.
